Taskfile.yml

```diff
@@ -88,12 +88,12 @@ tasks:
   export-cchain-block-range:
     desc: Export range of C-Chain blocks from source to target directory.
     vars:
-      SOURCE_BLOCK_DIR: '{{.SOURCE_BLOCK_DIR}}'
-      TARGET_BLOCK_DIR: '{{.TARGET_BLOCK_DIR}}'
+      BLOCK_DIR: '{{.BLOCK_DIR}}'
```

Comment:
This could be `BLOCK_DIR_SRC` as well since it's also copying, but in this case it's an export as opposed to import. Probably better to update to include the suffix.